dao-light/tests/utils.rs

```diff
@@ -0,0 +1,83 @@
+use codec::Encode;
+use dao_light_io::*;
+use ft_io::*;
+use gtest::{Program, RunResult, System};
+pub const MEMBERS: &'static [u64] = &[3, 4, 5, 6];
+pub const ZERO_ID: u64 = 0;
+
+pub fn init_fungible_token(sys: &System) {
+    sys.init_logger();
+    let ft = Program::from_file(
+        &sys,
+        "../target/wasm32-unknown-unknown/release/fungible_token.wasm",
+    );
+
+    let res = ft.send(
+        MEMBERS[0],
+        InitConfig {
+            name: String::from("MyToken"),
+            symbol: String::from("MTK"),
+        },
+    );
+
+    assert!(res.log().is_empty());
+    MEMBERS.iter().enumerate().for_each(|(_, member)| {
```

Comment:
Why do we enumerate and don't use the index?
```suggestion
    MEMBERS.iter().for_each(|member| {
```